app/scripts/metamask-controller.test.js

```diff
@@ -765,6 +765,56 @@ describe('MetaMaskController', () => {
     });
 
     describe('forgetDevice', () => {
+      let localMetaMaskController;
+
+      beforeEach(async () => {
```

Comment:
I'm wondering if this setup makes sense having in account the content of the test `should wipe all the keyring info`

Reply:
Maybe we consolidate the setup of both test since they are doing the same thing basically, but it can be in another PR